R/teal_transform_module.R

```diff
@@ -0,0 +1,179 @@
+#' Data module for `teal` transformers.
+#'
+#' @description
+#' `r lifecycle::badge("experimental")`
+#'
+#' `teal_transform_module` creates a shiny-module to transform data in a `teal` application.
+#'
+#' # Transforming `teal` module's input
```

Comment:
Might be good to mention the advantages of using the transform modules for these tasks. They come with error handling out of the box, etc. It's their biggest selling point and it's what stands them apart from just embedding app dev-supplied shiny modules into a module.